src/Utils.php

```diff
@@ -13,9 +13,7 @@ function get_pointer(string $current, string ...$parts): string {
 
 function get_function_name_from_function(mixed $function): string {
   $func_name = '';
-
-  if (\HHVM_VERSION_ID >= 42100) {
-    /*HH_FIXME[2049]*//*HH_FIXME[4107] These errors don't apply in hhvm > 4.21*/
```

Comment:
what's the actual minimum version of HHVM/Hack this project targets at present? I'm not seeing anything in composer.json, and from .travis.yml, it looks like 4.0 - so the fixmes need to stay

Reply:
If it's recent enough, using HHVM_VERSION_MAJOR and HHVM_VERSION_MINOR would be preferable to comparing HHVM_VERSION with `version_compare()`

Reply:
We should update this. The project will likely track the most recent version of HHVM we're using at Slack unless someone else updates. We're currently working towards 4.56. We can remove this check and I'll cut a new release for 4.56